spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I'd encourage you to add a test or remove this file. You could potentially use shoulda_matchers (which is already installed) to [test your `belongs_to` association](https://matchers.shoulda.io/docs/v4.1.2/Shoulda/Matchers/ActiveRecord.html#belong_to-instance_method) for example.

Reply:
Good point, thanks !